docs/samples/generic-host/AttributeApi/AttributeApi.csproj

```diff
@@ -10,7 +10,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\..\src\Hosting.CommandLine\McMaster.Extensions.Hosting.CommandLine.csproj" />
+    <ProjectReference Include="..\..\..\..\src\Hosting.CommandLine\McMaster.Extensions.Hosting.CommandLine.csproj" />
```

Comment:
FYI - moving this will break some of the docs. You'll also need to update docs/docs/advanced/generic-host.md to point to the new location.